libjlm/include/jlm/ir/hls/hls.hpp

```diff
@@ -0,0 +1,560 @@
+//
+// Created by david on 7/2/21.
+//
```

Comment:
Could we bring the headers for all newly added files into the same format as all the other headers, please?

Reply:
The headers have been replaced